frontend/src/index.jade

```diff
@@ -79,13 +79,29 @@ html
                 option(value="groupByRepos") Repo/Branch
                 option(value="groupByAuthors") Author
               label group by
-            .mui-select
-              select(v-model="filterSort")
-                option(value="totalCommits") Contribution
-                option(value="variance") Variance
-                option(value="displayName", v-if="filterGroupSelection !== 'groupByAuthors'") Author Name
-                option(value="searchPath", v-if="filterGroupSelection !== 'groupByRepos'") Repo/Branch Name
-              label sort by
+            .mui-select.sort-group
+              select(v-model="sortGroupSelection")
+                option(value="name", v-if="filterGroupSelection !== 'groupByRepos'") &uarr; name
+                option(value="name dsc", v-if="filterGroupSelection !== 'groupByRepos'") &darr; name
+                option(value="searchPath", v-if="filterGroupSelection !== 'groupByAuthors'") &uarr; repo/branch
+                option(value="searchPath dsc", v-if="filterGroupSelection !== 'groupByAuthors'") &darr; repo/branch
+                option(value="totalCommits", v-if="filterGroupSelection === 'groupByNone'") &uarr; contribution
+                option(value="totalCommits dsc", v-if="filterGroupSelection === 'groupByNone'") &darr; contribution
+                option(value="variance", v-if="filterGroupSelection === 'groupByNone'") &uarr; variance
+                option(value="variance dsc", v-if="filterGroupSelection === 'groupByNone'") &darr; variance
```

Comment:
just noticed this.

It seems that variance and totalCommits are only enabled when authors wasn't grouped. 
Was this the supposed functionality?

Reply:
Originally intended to continue off from #486, but I can implement it here as well. Is sorting by variance is taken at average?

Reply:
>Originally intended to continue off from #486, but I can implement it here as well. Is sorting by variance is taken at average?

I see. I guess we can continue in #486.